src/activities/associateGrade/AssociateGradeEntity.js

```diff
@@ -69,7 +70,17 @@ export class AssociateGradeEntity extends Entity {
 		return newGradeEntity && newGradeEntity.hasSubEntityByClass(Classes.activities.associateGrade.selectbox);
 	}
 
+	gradeItemId() {
+		return this._entity && this._entity.properties && this._entity.properties.gradeItemId;
+	}
+
+	originalGradeItemId() {
+		return this._entity && this._entity.properties && this._entity.properties.originalGradeItemId;
+	}
+
 	canEditGradebookStatus() {
+		if (!this._entity) return false;
```

Comment:
shouldn't be needed

Reply:
yeah, your return statement will return false.